components/button/button.css

```diff
@@ -0,0 +1,124 @@
+:root {
+  --primary-blue: #0066ff;
+  --primary-hover-blue: #0466c8;
+  --whitesmoke-color: whitesmoke;
+  --dark-theme-bg-color: #121212;
+  --secondary-dark-blue: #1d2d44;
+  --secondary-dark-blue-hover: #33415c;
+}
+
+body {
+  background-color: transparent;
+}
+
+.btn-container {
+  margin: 1rem;
+}
+
+.btn {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  border-radius: 5px;
+  gap: 0.4rem;
+  margin: 0.5rem;
+  padding: 0.8rem;
+  cursor: pointer;
+  box-shadow: 0px 3px 1px -2px RGB(0 0 0/20%), 0px 2px 2px 0px RGB(0 0 0/14%),
+    0px 1px 5px 0px RGB(0 0 0/12%);
+}
```

Comment:
you can write the RGB color function in lowercase, as the syntax is in lower case I guess.